### DISCUSSION TITLE
Do you get the supabase dashboard on the open source version ?

### DISCUSSION THREAD

**Discussion by AaronNGray** [supabase/supabase#1923] [Questions] [answered]
Do you get the supabase dashboard on the open source version ?

**Comment by soedirgo** [comment, marked as answer]
Hey @AaronNGray, the dashboard isn't in the local setup yet, we're taking steps towards this and have released [Supabase Grid](https://github.com/supabase/grid) which the dashboard is based on.

See also https://github.com/supabase/supabase/discussions/1001.

**Reply by AaronNGray** [reply]
Thanks, I like this GitHub Discussions, very cool thing for GitHub to have done :)

Can I get at the functionality using your cli for now ? And if so can you point me to any documentation please ?

**Reply by soedirgo** [reply]
> I like this GitHub Discussions

That makes two of us :)

> Can I get at the functionality using your cli for now ?

The CLI is for local development so it doesn't have the dashboard for now. It currently has limited functionality which is documented [here](https://supabase.io/docs/guides/self-hosting), though we're working on adding [migrations](https://supabase.io/blog/2021/03/31/supabase-cli#migrations) so you can move your database around. Stay tuned!

**Reply by AaronNGray** [reply]
> > Can I get at the functionality using your cli for now ?
> 
> The CLI is for local development so it doesn't have the dashboard for now. It currently has limited functionality which is documented [here](https://supabase.io/docs/guides/self-hosting), though we're working on adding [migrations](https://supabase.io/blog/2021/03/31/supabase-cli#migrations) so you can move your database around. Stay tuned!

I am planning on working locally I usually do for most projects, hence the line of questioning.
